token-lending/sdk/src/oracles.rs

```diff
@@ -23,6 +24,7 @@ const STALE_AFTER_SECONDS_ELAPSED: u64 = 120; // roughly 2 min
 
 pub enum OracleType {
     Pyth,
+    SbOnDemand,
```

Comment:
minor but would put this at the end. we dont serialize and store this enum anywhere but should append just to be safe